projects/documentation/content/guides/migration-guide.md

```diff
@@ -0,0 +1,132 @@
+
+<details>
+	<summary><strong>Package Versions</strong></summary>
+	<div class="table-container">
+		<table class="spectrum-Table spectrum-Table--sizeM">
+			<thead class="spectrum-Table-head">
+				<tr>
+					<th class="spectrum-Table-headCell">Package</th>
+					<th class="spectrum-Table-headCell">Version</th>
+				</tr>
+			</thead>
+			<tbody class="spectrum-Table-body">
+				{% for pkg in versions %}<tr class="spectrum-Table-row">
+					<td class="spectrum-Table-cell">
+						<a href="{{ pkg.dir }}">{{ pkg.name }}</a>
+					</td>
+					<td class="spectrum-Table-cell">
+						<a href="https://www.npmjs.com/package/{{ pkg.name }}/bundle/v/{{ pkg.version }}">{{ pkg.version }}</a>
+					</td>
+				<tr>{% endfor %}
+			</tbody>
+		</table>
+	</div>
+</details>
+
+### Using `lit@2.0` inside of `lit-html` and/or `LitElement`
+
+If you are currently using `lit-html@1.x` and/or `lit-element@2.x` within your project and are looking to consume these new package versions without updating to `lit` as well, you should generally have no issue. Manual migration may be required in the case that your application is consuming template literal versions of the icons available in `@spectrum-web-components/icons-ui` or `@spectrum-web-components/icons-workflow` directly from their default exports as follows:
+
+```js
+import { LitElement } from 'lit-element';
+import { html } from 'lit-html';
+import '@spectrum-web-components/icon/sp-icon.js';
+import { RedoIcon } from '@spectrum-web-components/icons-workflow';
+
+export class ElementWithicon extends LitElement {
+    render() {
+        return html`
+            // ...
+            <sp-icon>${RedoIcon()}</sp-icon>
+            // ...
+        `;
+    }
+}
+```
+
+In this case the `RedoIcon()` literal will be wrapped in the `html` template literal tag as supplied by `@spectrum-web-components/icons-workflow`. After this update, the version of this tag will no longer be compatible with the `html` template literal tag applied in the `render()` method of your custom element. You can correct this by applying your local value for `html` to the icon literals:
+
+```js
+import { html } from 'lit-html';
+import {
+    RedoIcon,
+    setCustomTemplateLiteralTag,
+} from '@spectrum-web-components/icons-workflow';
+
+setCustomTemplateLiteralTag(html);
+```
+
+Or, you can avoid running into issues like this in the future, while leveraging more performant icon delivery, by leveraging fully registered icon elements instead:
+
+```js
+import { LitElement } from 'lit-element';
+import { html } from 'lit-html';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-redo.js';
+
+export class ElementWithicon extends LitElement {
+    render() {
+        return html`
+            // ...
+            <sp-icon-redo></sp-icon-redo>
+            // ...
+        `;
+    }
+}
+```
+
+### Javascript re-exports
```

Comment:
Clarifies some changes that are not yet in #1401 but have been previously announced the the leads team.